src/cf-tables/usage.md

```diff
@@ -93,16 +94,13 @@ the `data-label` attribute for each cell.
 </table>
 ```
 
-### Table with small screen variation - "stacked with entry header"
 
-The `.table__entry-header-on-small` class in addition to `.table__stack-on-
-small` class changes the first column to be styled as an   entry header. This
-style requires both classes be added. Note that tables are not responsive
-without adding one of the small   screen classes. Also note that the `data-
-label` attribute is used to label each entry. Always include the `data-label`
-attribute for each cell.
+## Right-aligned cells.
```

Comment:
This is the only heading with a full stop, should it be there or should the others be updated to include it?